# Core configuration
comms_plugin: "labust::seatrac::SeatracSim"
controller_plugin: "labust::seatrac::USBLController"
listener_plugins: ["labust::seatrac::StatusListener",
                   "labust::seatrac::DataListener"]

# USBLController configuration
auto_mode: false
timeout: 10
transponders: [1,3]
transponder_names: ["surface","buddy"]


# Simulation parameters
sim_node_id: 2
sim_is_modem: true

